ms-docs/JBoss EAP on Azure.MD

```diff
@@ -27,40 +27,41 @@ These Quickstarts templates will show you how to deploy [JBoss Enterprise (EAP)]
 
 - **Start JBoss EAP as a Standalone Server** - the following command is how you start EAP service in Standalone Mode.
 
     `$EAP_HOME/bin/standalone.sh`
     
-    This startup script uses the EAP_HOME/bin/standalone.conf file, to set some default preferences, such as JVM options. Settings can be customized in this file. JBoss EAP uses the standalone.xml configuration file to start on Standalone mode by default, but can be started using a different mode. For details on the available standalone configuration files and how to use them, see the [Standalone Server Configuration Files](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html/configuration_guide/jboss_eap_management#standalone_server_configuration_files) section. To start JBoss EAP with a different configuration, use the --server-config argument. For example:
+    This startup script uses the EAP_HOME/bin/standalone.conf file to set some default preferences, such as JVM options. Settings can be customized in this file. JBoss EAP uses the standalone.xml configuration file to start on Standalone mode by default, but can be started using a different mode. For details on the available standalone configuration files and how to use them, see the [Standalone Server Configuration Files](https://access.redhat.com/documentation/red_hat_jboss_enterprise_application_platform/7.2/html/configuration_guide/jboss_eap_management#standalone_server_configuration_files) section. To start JBoss EAP with a different configuration, use the --server-config argument. For example:
     
     `$EAP_HOME/bin/standalone.sh --server-config=standalone-full.xml`
     
-    For a complete listing of all available startup script arguments and their purposes, use the --help argument or see the [Server Runtime Arguments](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html/configuration_guide/reference_material#reference_of_switches_and_arguments_to_pass_at_server_runtime) section.
+    For a complete listing of all available startup script arguments and their purposes, use the --help argument or see the [Server Runtime Arguments](https://access.redhat.com/documentation/red_hat_jboss_enterprise_application_platform/7.2/html/configuration_guide/reference_material#reference_of_switches_and_arguments_to_pass_at_server_runtime) section.

```

Comment:
The Link mentioned for **Server Runtime Arguments** is not accessible without en-us/
https://access.redhat.com/documentation/red_hat_jboss_enterprise_application_platform/7.2/html/configuration_guide/reference_material#reference_of_switches_and_arguments_to_pass_at_server_runtime